src/signals/incident/definitions/wizard-step-2-vulaan/openbaarGroenEnWater.ts

```diff
@@ -18,6 +18,16 @@ export const controls = {
         subcategory: 'eikenprocessierups',
       },
       label: 'Kies de boom waarin u de eikenprocessierupsen hebt gezien',
+      language: {
+        title: 'Locatie',
+        subTitle: 'Kies de boom op de kaart',
+        unregistered: 'De boom staat niet op de kaart',
+        unregisteredId: undefined,
+        objectTypeSingular: 'boom',
+        objectTypePlural: 'bomen',
+        submitSingular: 'Gebruik deze locatie',
+        submitPlural: 'Gebruik deze locatie',
```

Comment:
This prop can go; we don't support multiple objects

Reply:
We do need this prop. in the Selector: 
`          Zoom in om de {meta?.language?.objectTypePlural || 'objecten'} te zien
`